Tools/TestWebKitAPI/Tests/WebKitCocoa/ProcessSwapOnNavigation.mm

```diff
@@ -7906,7 +7906,7 @@ void configureCaptivePortalWKWebViewConfiguration(WKWebViewConfiguration *config
 
     delegate.get().decidePolicyForNavigationActionWithPreferences = ^(WKNavigationAction *action, WKWebpagePreferences *preferences, void (^completionHandler)(WKNavigationActionPolicy, WKWebpagePreferences *)) {
         EXPECT_FALSE(preferences._captivePortalModeEnabled);
```

Comment:
There are a bunch of references to _captivePortalModeEnabled that remain in this file and that should be updated to use the new API too.

Reply:
I'll do a more thorough search for instances of it.